src/core_plugins/metrics/public/components/vis_editor_visualization.js

```diff
@@ -87,9 +87,11 @@ class VisEditorVisualization extends Component {
   }
 
   componentDidUpdate() {
-    this._handler.update({
-      timeRange: this.props.timeRange
-    });
+    if (this._handler) {
+      this._handler.update({
+        timeRange: this.props.timeRange
+      });
+    }
```

Comment:
I am not sure if we don't lose a possible update here in a way, that we can end up with an old state. I think in case the handler is not defined yet, we should set a flag and update immediate after first rendering?